src/test/java/com/cloudant/tests/ViewsTest.java

```diff
@@ -552,6 +1004,35 @@ private Page checkPagesForward(int currentPage, int numberOfPages, Page page, bo
     }
 
     /**
+     * For use with multi-value key view tests.
+     *
+     * Check all the pages going forwards until we reach the last page. This assumes the given
+     * {@code page} is the first page of results.
+     *
+     * @param currentPage   the page number of the {@code page}.
+     * @param numberOfPages the number of pages to page forwards.
+     * @param page          the page from which to start paging forwards.
+     * @param descending    true if the view is in descending order, and false otherwise.
+     * @param docCount      the total number of documents in the view.
+     * @param docsPerPage   the maximum number of documents per page.
+     * @param view          the {@link View} object to use to perform the query (or null to create
+     *                         a new
+     *                      view for the query).
+     * @param viewCount     the current view in the array of multi-value key views. This value is
+     *                      used when view parameter is null.
+     * @return the last page in the view.
+     */
+    private Page checkPagesForwardMultiValueKey(int currentPage, int numberOfPages, Page page, boolean
+            descending, int docCount, int docsPerPage, View view, int viewCount) {
+        for (int i = 0; i < numberOfPages; ++i) {
```

Comment:
@mikerhodes this was the design in the original code.  I believe this was used because pages need to start at 1.